CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+---
+
+MOVE THIS SECTION TO THE CORRECT LOCATION
+
+### Misc. Updates
+
+- Model updates for the following services
+---AWSLambda
```

Comment:
```suggestion
```

Reply:
i've merged in AWSLambda, and not AWSIoT as that one is fairly large and requires some extra analysis to determine whether it is good to merge in